packages/datadog-plugin-mariadb/test/naming.js

```diff
@@ -0,0 +1,14 @@
+const { resolveNaming } = require('../../dd-trace/test/plugins/helpers')
+
+module.exports = resolveNaming({
+  outbound:
+  { v0: {
```

Comment:
The indentation of this whole object is really hard to follow.

Reply:
Looks like the indentation for `mysql` is fine, let's use that here too.

Reply:
A bit surprised eslint doesn't complain about that. 🤔

Reply:
Oof, that one sneaked out - these got refactored, indeed it's surprising there's no warning on it.